pkg/cmd/admin/registry/registry.go

```diff
@@ -74,7 +75,14 @@ type RegistryConfig struct {
 
 var errExit = fmt.Errorf("exit")
 
-const defaultLabel = "docker-registry=default"
+const (
+	defaultLabel = "docker-registry=default"
+	defaultPort  = 5000
```

Comment:
But it's [used as an integer](https://github.com/openshift/origin/pull/6439/files#diff-df3b6baf7e3b18ed4b1ff84012467504R241) as well. So I'd prefer it's native type.